draft-ietf-scitt-architecture.md

```diff
@@ -428,142 +425,62 @@ Verifiers can choose which Issuers they trust.
+The second set of entries are Signed Statements for additional domain-specific Registration Policy.
+The third set of entries are Signed Statements for Artifacts.
+From here on a Transparency Service is capable to check Signed Statements on registration via policy (that is at minimum key material and typically other Registration Policy) and is therefore in a reliable state to register Signed Statements about Artifacts or new Registration Policy.
 
-A Transparency Service MAY provide extra evidence that it is securely implemented and operated, enabling remote authentication of the hardware platforms and/or software TCB that run the Transparency Service.
-If present, this additional evidence MUST be recorded in the Append-only Log and presented on demand to Verifiers and Auditors.
-Examples for Statements that can improve trustworthy assessments of Transparency Services are RATS Conceptual Messages, such as Evidence, Endorsements, or corresponding Attestation Results (see {{-rats-arch}}).
+### Transparency Service Identity and Authenticity
 
-For example, consider a Transparency Service implemented using a set of replicas, each running within its own hardware-protected trusted execution environments (TEEs).
-Each replica MAY provide a recent attestation report for its TEE, binding their hardware platform to the software that runs the Transparency Service, the long-term public key of the service, and the key used by the replica for signing Receipts.
-This attestation evidence can be supplemented with Receipts for the software and configuration of the service, as measured in its attestation report.
-
-### Configuration
-
-The Transparency Service records its configuration in the Append-Only Log using Transparent Statements with distinguished media type `application/scitt-configuration`.
-
-The registration policy for statements with the media type suffix (`+<format>` is implementation-specific.
-The implementation SHOULD document them, for example defining the Issuers authorized to register configuration Signed Statements.
-
-The Transparency Service is configured by the last Transparent Statement of this type.
-The Transparency Service MUST register a Signed Statement that defines its initial configuration before registering any other Signed Statement.
-The Transparency Service MAY register an additional Signed Statement that updates its configuration.
-
-The Transparency Service provides an endpoint that returns the Transparent Statement that describes its current configuration.
-
-The configuration `reg_info` SHOULD include a secure version number and a timestamp.
-
-The sample configuration payload uses the CDDL {{-CDDL}}
-
-~~~ cddl
-Signature_Algorithms = [ int ]
+Every Transparency Service MUST have a public service identity that is associated with public/private key pairs for signing Receipts on behalf of the service.
+In particular, this service identity must be known by Verifiers when checking a Receipt's signature.
```

Comment:
A note that the section marked for deletion here is captured later [here](https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/pull/179/files#diff-585fb424519b79cf000445f3425fa56d328cbaca5d2a622740ddc4e5ca91dbe1R888)

Reply:
I was just moving security considerations to that section, I did not attempt to actually improve the security considerations.